db-server/test/backend/db_tests.js

```diff
@@ -1204,10 +1207,14 @@ module.exports = function (config, DB) {
         return db.account(accountData.uid)
           .then((account) => {
             assert.ok(account, 'account exists')
+            accountData.verifierSetAt = now + 1
             return db.resetAccount(accountData.uid, accountData)
           })
           .then(() => db.account(accountData.uid))
-          .then((account) => assert.ok(account, 'account exists'))
+          .then((account) => {
+            assert.ok(account, 'account exists')
+            assert.equal(account.profileChangedAt > account.createdAt, true, 'profileChangedAt updated')
```

Comment:
IIUC you could also check here that `profileChangedAt` is requal to `verifierSetAt`?